src/components/dashboard/Dashboard.vue

```diff
@@ -3,10 +3,10 @@
     <div class="container--chart-panels">
       <div class="container--value-panel">
         <div class="container--panel">
-          <value-panel title="Current Circulating Supply" :value="circulatingSupply" />
+          <circulating-panel :symbol="chainInfo.tokenSymbol" />
         </div>
         <div class="container--panel">
-          <value-panel title="Total Supply" :value="totalSupply" />
+          <rewards-panel :symbol="chainInfo.tokenSymbol" />
```

Comment:
if we plan to add this view on the dApps Staking page at some point, I don't think it makes sense to add this on the dashboard page now?
What are your thoughts @Kahonnohak ?

Reply:
Yes this needs to be on dApp Staking. i just commented on Slack. The question is how much we are adding before we can move to V2. V1 just got "your rewards' section which is appropriate for showing the total earned rewards however the space is tight and will need additional frontend development which is pitty..) Let me think.

The total Earned display makes good encouragement to users indeed and is amazing, however, the ideal goal is to have the total earned per project not all-time.

Reply:
Fixed
![image](https://user-images.githubusercontent.com/92044428/167285989-d1ba4ffe-401b-4a1f-b398-1d1be4816eab.png)